pkg/asset/machines/worker.go

```diff
@@ -223,6 +223,13 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 			}
 			machineConfigs = append(machineConfigs, ignFIPS)
 		}
+		if len(pool.Workloads) > 0 {
```

Comment:
Don't put this code in here yet, since we don't support it.

Reply:
Good point.